src/DankCord/DankCord.py

```diff
@@ -1,17 +1,16 @@
-import datetime
-import json
-import time
-from string import printable
-from typing import Optional
-
+import datetime, json, time, orjson
 import faster_than_requests as requests
-import orjson
+
+from rich import print
+from string import printable
+from typing import Literal, Optional, Union
 from pyloggor import pyloggor
+from threading import Thread
```

Comment:
Although PEP8 is supposed to "beautify" code, I do not find it beautiful in many ways, the imports are an example of that. Following PEP8 is not necessary, the code still runs perfectly, so it's just a matter of preference at this point. Plus, importing a certain function from a whole module is faster than importing the whole module, especially since we're only using `Thread` out of the whole `threading` module.